### PR TITLE
Add developer docs for integrating the pollination viewer

### PR DESCRIPTION
**Pull request by NGimbal**
@mostaphaRoudsari - let me know what the best way to merge this in is.

### CODE REVIEW

File: pollination-apps/app-developers/integrating-the-pollination-viewer.md
Comment (mostaphaRoudsari) [inline pollination-apps/app-developers/integrating-the-pollination-viewer.md:18]: This is great that you are providing the link. You probably want to remove these two and add them to `.gitignore`:

image

File: pollination-apps/app-developers/integrating-the-pollination-viewer.md
Comment (mostaphaRoudsari) [inline pollination-apps/app-developers/integrating-the-pollination-viewer.md:57]: I might be wrong but you usually want to leave a blank line between the lines in a markdown document. Otherwise, they may not be rendered correctly on some platforms.

File: pollination-apps/app-developers/integrating-the-pollination-viewer.md
Comment (mostaphaRoudsari) [inline pollination-apps/app-developers/integrating-the-pollination-viewer.md:20]: Should we also provide a link to a sample `vtkjs` file? I would think that makes it easier for people to get started.

I also think your sample app will be more playful if it loads a sample file when I open the page.

File: pollination-apps/app-developers/integrating-the-pollination-viewer.md
Comment (mostaphaRoudsari) [inline pollination-apps/app-developers/integrating-the-pollination-viewer.md:92]: Nice! ✨

**Review by mostaphaRoudsari — APPROVED** [review]
See my comments! Overall it looks good to be merged. I think we want to coordinate the folder structure between this PR and the other one by devang. I like having the extra `app-developers` subfolder but I leave this to @jv-ladybugtools to decide. We can also move this document around as needed after it is merged.

**Comment by jv-ladybugtools** [comment]
Yes, I’d rather reorg as required once merged because it’s not always a clean transfer between Git Hub and GitBook. 

JANKI A VYAS, AIA, LEED AP BD+C, O+M (she/her)
Principal @ KARMA co/lab
Lecturer @ University of Pennsylvania

File: pollination-apps/app-developers/integrating-the-pollination-viewer.md
Comment (NGimbal) [reply pollination-apps/app-developers/integrating-the-pollination-viewer.md:20]: How can we provide a sample vtkjs file for someone? Can that be committed in the repo or is there a better way to do that?

**Pull request merged**
